shell/platform/tizen/tizen_evas_gl_helper.h

```diff
@@ -0,0 +1,687 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This file has same content as tizen's file(Evas_GL_GLES3_Helpers.h).
+// https://docs.tizen.org/application/native/api/mobile/6.0/group__Evas__GL__GLES3__Helpers.html
```

Comment:
I'm not sure, but...I doubt this is the best option. Do we have to do this?
(If I'm not mistaken... is this change added recently?)

Reply:
Well.. I didn't quite understand the your point... 
Does your intention mean we shouldn't use that macro in renderers? 
I think that this macro could be used because it would improve readability and have no dependencies. 
Let me know your concerns & I'll check it!

Reply:
My intention is that I want to avoid keeping a copy of Evans_GLES3_Helpers.h in engine source tree

Reply:
It is ideal to remove the file from our source tree. Is there any way to do so?

Reply:
> It is ideal to remove the file from our source tree. Is there any way to do so?

Remove macro at evasgl renderer? @bbrto21  said to include the path of the tizen sdk... To be honest, I don't know...
IMO, We should make a choice between ``duplicate`` or ``removing macro``. (I can't think of any good ideas... :cry: )